sections/docker.zsh

```diff
@@ -24,13 +24,33 @@ spaceship_docker() {
 
   spaceship::exists docker || return
 
+  # Better support for docker environment vars: https://docs.docker.com/compose/reference/envvars/
+  # Use COMPOSE_PATH_SEPARATOR or colon as default
+  local separator=":"
+  [[ ! -z $COMPOSE_PATH_SEPARATOR ]] && separator=$COMPOSE_PATH_SEPARATOR
```

Comment:
Why not use [paramater expansion](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Parameter-Expansion) to assign variable like ,

```diff
-  local separator=":"
-  [[ ! -z $COMPOSE_PATH_SEPARATOR ]] && separator=$COMPOSE_PATH_SEPARATOR

+  local separator="${COMPOSE_PATH_SEPARATOR=":"}"
 ```

Reply:
sorry, that's my first time coding in zsh language. I'll do that!